ReactNativeClient/lib/fs-driver-rn.js

```diff
@@ -156,6 +156,12 @@ class FsDriverRN extends FsDriverBase {
 		handle.offset += length;
 		return output ? output : null;
 	}
+
+	async fileSize(filePath) {
+		const stats = await RNFetchBlob.fs.stat(filePath);
+		return stats.size;
+	}
+
```

Comment:
Same, you don't need to create a new method, you can simply use the existing `stat()` method.